android-project/app/src/main/java/org/libsdl/app/SDLActivity.java

```diff
@@ -2116,6 +2117,48 @@ static class SDLFileDialogState {
         int requestCode;
         boolean multipleChoice;
     }
+    
+    /**
+     * This method is called by SDL using JNI.
+     */
+    public static String getPreferredLocales() {
+        String result = "";
+        if (Build.VERSION.SDK_INT >= 24 /* Android 7 (N) */) {
+            LocaleList locales = LocaleList.getAdjustedDefault();
+            for (int i = 0; i < locales.size(); i++) {
+                if (i != 0) result += ",";
+                result += formatLocale(locales.get(i));
+            }
+        }
+        else if (mCurrentLocale != null) {
```

Comment:
```suggestion
        } else if (mCurrentLocale != null) {
```
SDL style is "else on the same line", can you use that style in your PR?